packages/fxa-content-server/app/scripts/views/sign_in_unblock.js

```diff
@@ -64,6 +130,13 @@ const View = FormView.extend({
       // re-throw, it'll be displayed at a lower level.
       throw error;
     }
+
+    if(AuthErrors.is(error, 'INVALID_UNBLOCK_CODE')) {
```

Comment:
This line would not get executed since it is below the `if...else` block above which throws an error. You should be able to put in the in else block.